prover/prover_dal/migrations/20240419102606_add_changes_for_recursion_tip.up.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS scheduler_dependency_tracker_fri;
+
+CREATE TABLE IF NOT EXISTS recursion_tip_witness_jobs_fri (
+    l1_batch_number           BIGSERIAL PRIMARY KEY,
```

Comment:
nit: I think BIGINT is more appropriate here

Reply:
`BIGSERIAL` allows you to insert row without `l1_batch_number` and it will be assigned automatically and most probably will be incorrect